TEMPLATE.md

```diff
@@ -1,4 +1,7 @@
-# <Name of Feature>
+# {Name of Feature}
+
+- Owner: {Your Name} {you@example.com}, {@you} on the Elm Slack
```

Comment:
I'm not thrilled with the idea of owners, I was imagining something more like a wiki. Would you mind changing it to *author*?